source/services/index.js

```diff
@@ -1,30 +1,37 @@
-'use strict';
-
 /**
  * Social network services
  */
+import Service from '../service';
+
+import facebook from './facebook';
+import gplus from './gplus';
+import odnoklassniki from './odnoklassniki';
+import pinterest from './pinterest';
+
+import svg from '../svg.js';
+
+import telegram from './telegram';
```

Comment:
Let’s group social imports here together, without `svg` breaking in? (Alphabetical sorting in all other places is ❤️, actually.)